README.md

```diff
@@ -983,6 +989,58 @@ Runtime options:
 Generates (but does not run) a migration to add a versions table.  Also generates an initializer file for configuring PaperTrail
 ```
 
+##### paper_trail:update_sti
+
+The second generator, `paper_trail:update_sti`, also writes but does not run a migration file
+which scans existing data in the versions table to identify all use of STI. Upon finding each
+entry which refers to an object from a subclassed model, `item_type` is updated to reflect
+the subclass name, providing full compatibility with how the `versions` association works
+in >= v9.2. In order to more quickly update a large volume of version records, updates are
+batched to affect 100 records at a time.
```

Comment:
Excellent docs. And, I see this section is referenced from the changelog. Can we make that into a link?

Reply:
Thanks much.  Have put in a link simply to 5.c.  Can be made more specific perhaps.